src/openMVG/sfm/pipelines/localization/SfM_Localizer_Single_3DTrackObservation_Database.cpp

```diff
@@ -80,16 +80,22 @@ namespace sfm {
   {
     if (sfm_data_ == nullptr)
     {
+      std::cerr << std::endl
+        << "The internal sfm_data structure is not initialized!" << std::endl;
       return false;
     }
 
     matching::IndMatches vec_putative_matches;
     if (!matching_interface_.Match(0.8, query_regions, vec_putative_matches))
     {
+      std::cerr << std::endl
+        << "Matching failed!" << std::endl;
       return false;
     }
 
-    std::cout << "#3D2d putative correspondences: " << vec_putative_matches.size() << std::endl;
+    std::cout << "Number of 3D2d putative correspondences found: " 
```

Comment:
FYI, I use # to mean cardinal ("number of"), this tip allow to make more compact log.